lm_eval/tasks/unitxt/tasks.classification.multi_class.yaml

```diff
@@ -0,0 +1,26 @@
+group:
+- unitxt
+task: TASK_PLACEHOLDER
```

Comment:
Task templates don't require placeholders, but it may help a user to remind them what needs to be filled in if they want to include this template. I think `task` and `dataset_name` could be commented out so users can see them.